### PR TITLE
[BB-2186] Support special vacation

### PR DESCRIPTION
**Pull request by Agrendalath**
## What it is
This adds support for scheduling [special vacation](https://handbook.opencraft.com/en/latest/vacation/#special-vacation) - reduced time off.

## How it works
Due to many mistakes and oversights we relaxed regex to treat any event starting with member's name as the vacation. Additionally, if it's followed with any word and then a number after which comers letter "h", it is treated as reduced time off - **the provided number of hours indicates user's availability per day**. Some working examples:
- Piotr off
- Piotr away
- Piotr reduced 4 h only
- Piotr reduced time off (4h/day)

## Testing instructions 
1. Set `.env` file (you can find it in a comment on the ticket).
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Create superuser with `docker-compose -f local.yml exec django python manage.py createsuperuser`
1. Run migrations with `docker-compose -f local.yml migrate`.
1. Navigate to `frontend` directory and run `npm i && npm start`.
1. Log in.
1. Create some test events and see if they appear on the dashboard in `Vacation` column.

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -607,4 +607,4 @@
 
 # Pool size for making parallel API requests
 MULTIPROCESSING_POOL_SIZE = env.int("MULTIPROCESSING_POOL_SIZE", 32)
-MULTIPROCESSING_TIMEOUT = env.int("MULTIPROCESSING_TIMEOUT", 32)
+MULTIPROCESSING_TIMEOUT = env.int("MULTIPROCESSING_TIMEOUT", 128)
```
Comment (Agrendalath) [inline config/settings/base.py]: It's just an unrelated side change, as 32 is a low default value.

File: config/settings/base.py [hunk 2]
```diff
@@ -539,7 +539,7 @@
 # By default we're using f"{name} off" format, which works fine with `name` being at least user's first name.
```
Comment (giovannicimolin) [inline config/settings/base.py]: @Agrendalath Can you update this comment to reflect on the new regex behaviour?

Comment (Agrendalath) [reply config/settings/base.py]: Good catch!
